src/main/scala/com/databricks/labs/overwatch/ApiCall.scala

```diff
@@ -150,6 +150,10 @@ class ApiCall(env: ApiEnv) extends SparkSessionWrapper {
         case "instance-pools/list" => "instance_pools"
         case "instance-profiles/list" => "instance_profiles"
         case "workspace/list" => "objects"
+        case "libraries/all-cluster-statuses" => "statuses"
```

Comment:
why APICall??? we should be using ApiCallV2 for all of our enhancement.

Reply:
As per the issue description we want to use APICallV2 for Job runs

Reply:
@aman-db -- I did not add this. I'm not sure which branch it came in on

Reply:
@Sriram-databricks removed the legacy code from APICall.scala